Moderation-Policy.md

```diff
@@ -1,154 +1,306 @@
+Organization and all Node.js Working Groups.
+
+Individual Working Groups and Top Level Projects may adopt an alternative
+Moderation Policy for any repository under their stewardship so long as:
+
+* The Moderation Policy is openly documented as part of the Working Group or
+  Project Charter and;
+* Includes provisions for clearly and openly documenting Moderation actions
+  taken.
+
+If a particular repository can be considered to fall under the stewardship of
+multiple Working Groups (or Top Level Projects) that have adopted different
+Moderation Policies, they can choose to either:
+
+* Decide between themselves which Moderation Policy will be in effect for the
+  repository in question, or
 * Ask the TSC to determine which Moderation Policy should apply.
 
-Any alternative Moderation Policy used for a given repository must be included in the root directory of the repository using the `Moderation-Policy.md` filename. If a repository does not contain a `Moderation-Policy.md` file, then this default policy applies.
-
-### Terms
-
-* *Collaborator* refers to any individual with configured write (commit) permissions to any Node.js GitHub organization repository *other than the Moderation Repository*. See [GitHub's access permissions documentation](https://help.github.com/articles/what-are-the-different-access-permissions/) for more information.
-* *TSC* refers to the [Node.js Technical Steering Committee](https://github.com/nodejs/node#tsc-technical-steering-committee).
-* *Post* refers to the content and titles of any issue, pull request, comment or wiki page.
-* *Moderate* refers to the act of modifying the content and title of, or deleting, any Post for the purpose of correcting or addressing Code of Conduct violations.
-* *Remove* refers to the act of removing the configured write (commit) permissions for an individual Collaborator's GitHub account from *all* Node.js GitHub Organization repositories as well as removing the account from the Node.js GitHub Organization membership.
-* *Ban* refers to the act of blocking an individual GitHub account from any further participation in the Node.js GitHub Organization.
+Any alternative Moderation Policy used for a given repository must be included
+in the root directory of the repository using the `Moderation-Policy.md`
+filename. If a repository does not contain a `Moderation-Policy.md` file, then
+this default policy applies.
+
+## Terms
+
+* *Collaborator* refers to any individual with configured write (commit)
+  permissions to any Node.js GitHub organization repository *other than the
+  Moderation Repository*. See [GitHub's access permissions documentation][] for
+  more information.
+* *TSC* refers to the [Node.js Technical Steering Committee][].
+* *CommComm* refers to the [Node.js Community Committee][].
+* *Post* refers to the content and titles of any issue, pull request, comment
+  or wiki page.
+* *Moderate* refers to the act of modifying the content and title of, or
```

Comment:
It is not clear to me whether self-moderation is included or excluded so I suggest either:
Moderation is when someone else edits your posts:
```markdown
*Moderate* refers to the act of modifying the content and title of, or
deleting, any Post made by another person for the purpose of correcting
or addressing Code of Conduct violations.
```
Or, merely the act of being called out is considered moderation:
```markdown
*Moderate* refers to the act of modifying the content and title of, or
deleting any Post, or requesting the author of said post to perform these actions,
for the purpose of correcting or addressing Code of Conduct violations.
```